start-vs.cmd

```diff
@@ -0,0 +1,44 @@
+@echo off
+setlocal enabledelayedexpansion
+
+:: This command launches a Visual Studio solution with environment variables required to use a local version of the .NET Core SDK.
```

Comment:
💡 I recommend removing this file. If Visual Studio does not support opening this solution directly, the solution should be corrected to comply with Visual Studio's practices.

Reply:
This workaround misleads developers working on internal versions of Visual Studio into thinking the product is working. We _need_ this to be an acute pain point until such time as we deliver a working product.

Reply:
@sharwell if you know how to get VS to use the local .NET copy please share. If not, and you think that VS should support this scenario, then please open an issue and I'll be happy to chime in.

Making WinForms/WPF devs daily life difficult is _not_ the answer to making VS better.